ext/pdo/pdo_dbh.c

```diff
@@ -699,6 +699,11 @@ static bool pdo_dbh_attribute_set(pdo_dbh_t *dbh, zend_long attr, zval *value) /
 {
 	zend_long lval;
 
+	if (!dbh->methods->set_attribute) {
```

Comment:
Err, doesn't that now fail also for attributes provided by PDO itself, like ATTR_ERRMODE? I don't think that was the intention here.

Reply:
Indeed, that's my mistake, I'll drop that commit.